AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DepthOfFieldAdded.py

```diff
@@ -175,29 +326,404 @@ def AtomEditorComponents_DepthOfField_AddedToEntity():
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('Enable Depth of Field')) is True)
+
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('Enable Depth of Field'), False)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.enable_depth_of_field_off,
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('Enable Depth of Field')) is False)
+
+        # Enable Depth of Field again for subsequent tests.
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('Enable Depth of Field'), True)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.enable_depth_of_field_on,
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('Enable Depth of Field')) is True)
+
+        # 26. Quality Level can be set to 0 and back to 1.
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('Quality Level'), 0)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.quality_level_set_to_0,
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('Quality Level')) == 0)
+
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('Quality Level'), 1)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.quality_level_set_to_1,
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('Quality Level')) == 1)
+
+        # 27. Aperture F can be set to 0 and back to 0.5.
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('Aperture F'), 0)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.aperturef_override_set_to_0,
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('Aperture F')) == 0)
+
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('Aperture F'), .5)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.aperture_f_set_to_five_tenths,
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('Aperture F')) == .5)
+
+        # F Number Note - Discussed it with the dev owner, and we determined this does not need to be tested.
+
+        # 28a. Enable Auto Focus can be turned off.
+        # Turn off Auto Focus to unlock Focus distance setting
```

Comment:
on comment line is enough. these say the same thing essentially or can be combined

Reply:
Disagree, this explains why there is an a and b version of the step